### PR TITLE
Dropdown => TypeScript

### LINKED ISSUE #188: Convert Dropdown to TypeScript

### PR DESCRIPTION
**Pull request by mdespuits**
Fixes #188

### CODE REVIEW

File: src/components/EasyDropdown/EasyDropdown.test.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { act } from 'react-dom/test-utils';
 
 import Button from '../Button';
 
-import { Menu } from '../Dropdown';
+import Dropdown from '../Dropdown';
```
Comment (pixelbandito) [inline src/components/EasyDropdown/EasyDropdown.test.js:7]: Why the new import without corresponding new usage?

File: src/components/EasyDropdown/index.js [hunk 2]
```diff
@@ -159,7 +159,7 @@ EasyDropdown.propTypes = {
       onClick: PropTypes.func,
     })
   ),
-  menuProps: PropTypes.shape({ ...Menu.propTypes }),
+  menuProps: PropTypes.object,
```
Comment (pixelbandito) [inline src/components/EasyDropdown/index.js:162]: Is this component coming up for TS-ification soon?

File: src/components/EasyDropdown/EasyDropdown.test.js (hunk 1 above)
Comment (pixelbandito) [reply src/components/EasyDropdown/EasyDropdown.test.js:7]: Guess this should be a change request, but it's minor, so 👍

File: src/components/EasyDropdown/index.js (hunk 2 above)
Comment (mdespuits) [reply src/components/EasyDropdown/index.js:162]: Yes, I have a branch for it I believe... It's coming, I promise

File: src/components/EasyDropdown/EasyDropdown.test.js (hunk 1 above)
Comment (mdespuits) [reply src/components/EasyDropdown/EasyDropdown.test.js:7]: Updated! 👍

**Pull request merged**